libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverviewChartPivot.tsx

```diff
@@ -25,7 +25,9 @@ import { RegressionDistributionChart } from "./RegressionDistributionChart";
 import { getSelectableMetrics } from "./StatsTableUtils";
 
 interface IModelOverviewChartPivotProps {
-  chartCohorts: ErrorCohort[];
+  allCohorts: ErrorCohort[];
+  selectedChartCohorts: number[];
+  isDatasetBasedCohorts: boolean;
```

Comment:
This doesn't quite sound right since it's plural. `showDatasetBasedCohorts`?

Reply:
done.